tests/devicetestcase.py

```diff
@@ -33,7 +33,8 @@ def setUpClass(cls):
             util.dd("/dev/zero", cls.backing_store, 256, bs="1M")
             cls.device = util.loop_create(cls.backing_store)
 
-        util.mkfs(cls.device)
+        cls.fs = os.getenv('TEST_FS') if os.getenv('TEST_FS') else "ext4"
```

Comment:
I'd suggest using the default value in the `os.getenv()` method

Reply:
Good idea. Applied.